app/models/world_location.rb

```diff
@@ -13,23 +13,18 @@ class WorldLocation < ApplicationRecord
            source: :document
   has_many :worldwide_organisation_world_locations, dependent: :destroy
   has_many :worldwide_organisations, through: :worldwide_organisation_world_locations
-  has_many :offsite_links, as: :parent
 
-  has_many :featured_links, -> { order(:created_at) }, as: :linkable, dependent: :destroy
-  accepts_nested_attributes_for :featured_links, reject_if: ->(attributes) { attributes["url"].blank? }, allow_destroy: true
-
-  include Featurable
+  has_one :world_location_news
```

Comment:
Really like how this cuts down the complexity of the model. I'm assuming the featured/offsite links were only used in the previously coupled world location news bit of this

Reply:
Yes, they were only used for the news part.  The rest of the `WorldLocation` didn't use them at all.